src/BSPTree.h

```diff
@@ -81,7 +93,7 @@ class CBSPTree
 		CBoundingBox& lBox = splitBoxes.first;
```

Comment:
It looks like the last problem was not done: splitDim and splitVal are unchanged